onchain/rollups/contracts/dapp/CartesiDApp.sol

```diff
@@ -11,9 +11,11 @@ import {Bitmask} from "@cartesi/util/contracts/Bitmask.sol";
 
 import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
 import {ERC721Holder} from "@openzeppelin/contracts/token/ERC721/utils/ERC721Holder.sol";
-import {ERC1155Holder} from "@openzeppelin/contracts/token/ERC1155/utils/ERC1155Holder.sol";
+import {ERC1155Holder, ERC1155Receiver} from "@openzeppelin/contracts/token/ERC1155/utils/ERC1155Holder.sol";
```

Comment:
in the newer versions of openzeppelin, `ERC1155Receiver` is merged into `ERC1155Holder`, and thus `ERC1155Receiver` is deprecated.